### PR TITLE
more segments for pal

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 650 functions (+2.51%) on `pal`.

File: src/main_loop.c [hunk 1]
```diff
@@ -5,6 +5,24 @@
 #include "sprite.h"
 #include "overlay.h"
 
+#if VERSION_PAL
```
Comment (marijnvdwerf) [inline src/main_loop.c:8]: Old stuff from working on `main` segment

File: src/state_battle.c [hunk 2]
```diff
@@ -6,6 +6,12 @@
 #include "battle/battle.h"
 #include "model.h"
 
+#if VERSION_PAL
```
Comment (marijnvdwerf) [inline src/state_battle.c:9]: Old stuff from working on `main` segment

File: src/world/world.c [hunk 3]
```diff
@@ -8,6 +8,12 @@
 #include "model.h"
 #include "gcc/string.h"
 
+#if VERSION_PAL
```
Comment (marijnvdwerf) [inline src/world/world.c:11]: Old stuff from working on `main` segment

File: ver/pal/splat.yaml [hunk 4]
```diff
@@ -34,6 +34,38 @@ segments:
...
   - [0x40, bin]
 
+  - name: evt
+    type: code
+    dir: evt
+    start: 0xE61D0
+    vram: 0x802C6000
+    # follows_vram: world_use_item # TODO this causes the vram to be wrong
+    subsegments:
+    - [auto, c, script_list]
```
Comment (marijnvdwerf) [inline ver/pal/splat.yaml:44]: Can we decide on a preferred list style?

File: ver/pal/undefined_syms_auto.txt [hunk 5]
```diff
@@ -20,3 +20,4 @@ D_B0000000 = 0xB0000000;
 D_B0000008 = 0xB0000008;
 D_B0000010 = 0xB0000010;
 fx_sun_undeclared = fx_sun;
+sprite_shading_profiles_data_ROM_START = sprite_shading_profiles_ROM_START + 0x1D0;
```
Comment (nanaian) [inline ver/pal/undefined_syms_auto.txt:23]: 🤨

Comment (ethteck) [reply ver/pal/undefined_syms_auto.txt:23]: this exists in the US undefined syms too. not ideal though, yeah

**Pull request merged**